pkg/jaeger/jaeger_event.go

```diff
@@ -0,0 +1,98 @@
+package jaeger
+
+import (
+	"strconv"
+	"time"
+
+	jaegerModel "github.com/jaegertracing/jaeger/model"
+
+	"github.com/pavolloffay/jaeger-local-storage-poc/pkg/model"
+)
+
+// Event The jaeger span version of the event
+type Event struct {
+	tenant    string
+	timestamp time.Time
+	fields    []*model.Field
+}
+
+// NewEvent creates an event from a Jaeger span
+func NewEvent(span *jaegerModel.Span) model.Event {
```

Comment:
Is there any architecture diagram showing why this interface is needed how is it related to other parts of the storage?

Reply:
No diagram currently, although would be good to have one I guess. The event interface just abstracts away from the details of the jaeger span - so only exposes the necessary details (e.g. list of fields primarily) that the store requires.